crates/forge_domain/src/workflow.rs

```diff
@@ -7,6 +7,19 @@ use serde_json::Value;
 
 use crate::{Agent, AgentId};
 
+#[derive(Debug, Clone, Serialize, Deserialize, Merge, Setters)]
+#[setters(strip_option, into)]
+pub struct ModeConfig {
+    #[merge(strategy = crate::merge::std::overwrite)]
+    pub name: String,
```

Comment:
Add a custom serde verifier for name such that is always alphabets, and numbers and underscores. Nothing else.